EasyPost/Resource.cs

```diff
@@ -64,11 +67,11 @@ public void Merge(object source)
 
             switch (value)
             {
-                case IResource resource:
+                case Resource resource:
```

Comment:
What's with the `IResource` to `Resource` change?

Reply:
IResource was an interface, I'm assuming added for reusability way-back-when, but it's not being used anywhere except here, and since it's an interface, it's interfering with the ability to change the access limit on some functions.

TL;DR old unnecessary code that was removed.